tck/faces40/inputFile/src/test/java/ee/jakarta/tck/faces/test/servlet50/inputfile/Spec1555IT.java

```diff
@@ -48,7 +49,7 @@
 
 import jakarta.faces.component.html.HtmlInputFile;
 
-@RunWith(Arquillian.class)
+@RunWith(BaseArquilianRunner.class)
```

Comment:
@werpu  As this is tagged with 4.0.1, could you undo this change?  It should be done a separate PR.

Reply:
I can undo it but then the Spec1555IT will be triggered always. That means the test will fail on myfaces because it triggers HTMLUnit with test.selenium=true

Reply:
I think Vlad may have tagged the wrong file here.  The BeanValidation test is the one that would want to be in a separate PR -- tck/faces40/beanValidation/src/test/java/ee/jakarta/tck/faces/test/servlet50/beanValidation/Issue5171IT.java

That way 4.0.1 changes can be siloed from changes to new tests.

Reply:
Oops sorry about that! 😞

Reply:
Ok I will isolate the bean validation change into a separate non 4.0.1 pr later today!
Thanks for the update!

Reply:
Hi I have isolated the BeanValidation fix in https://github.com/jakartaee/faces/pull/1797 the changes on the test are dropped from the this pull request!